.github/workflows/build.yml

```diff
@@ -139,21 +139,3 @@ jobs:
         if: github.event_name != 'pull_request'
         run: |
           echo "${{ toJSON(steps.push.outputs) }}"
-
-  copr-build:
```

Comment:
Don't know the history, but why is this getting removed?

Reply:
it was originally supposed to be a github action that would automatically tell a COPR to rebuild, it was abandoned a while ago and now the COPR automatically rebuilds on new releases